libs/stats/odc/stats/_cli_generate_mosaic.py

```diff
@@ -46,7 +50,7 @@ def xr_to_mem(xx, client):
     )
 
 
-def save(xx, location, product_name, verbose):
+def save(xx, location, product_name, verbose, creds=None, rgb_bands=None):
```

Comment:
Is this function still used?

Reply:
yeah it's no longer used, but I thought I'd keep it in here for now just in case there's some special RGB stuff that's needed for geomedian. Do you know if anyone is using this?

Reply:
(if not I'll go ahead and delete it)

Reply:
I don't think anyone uses this but check with Nikita before deleting it.

Reply:
just checked with Nikita and it doesn't affect her stuff :)

Reply:
Sounds like this isn't used then, might as well remove it.